src/TendoPay/API/Verification_Endpoint.php

```diff
@@ -52,15 +52,18 @@ public function verify_payment( \WC_Order $order, array $data ) {
 		$disposition                  = $data[ Constants::DISPOSITION_PARAM ];
 		$tendo_pay_transaction_number = $data[ Constants::TRANSACTION_NO_PARAM ];
 		$verification_token           = $data[ Constants::VERIFICATION_TOKEN_PARAM ];
+		$tendopayUserId               = $data[ Constants::TENDOPAY_USER_ID ];
 
 		$verification_data = [
-			Constants::ORDER_ID_PARAM           => (string) $order->get_id(),
-			Constants::ORDER_KEY_PARAM          => $order->get_order_key(),
-			Constants::DISPOSITION_PARAM        => $disposition,
-			Constants::VENDOR_ID_PARAM          => (string) $gateway_options[ Gateway::OPTION_TENDOPAY_VENDOR_ID ],
-			Constants::TRANSACTION_NO_PARAM     => (string) $tendo_pay_transaction_number,
-			Constants::VERIFICATION_TOKEN_PARAM => $verification_token
-		];
+            Constants::ORDER_ID_PARAM           => (string) $order->get_id(),
```

Comment:
The plugin's code style rules were set to meet the standard WP code style rules. Therefore indentation should be added using tabs, not spaces (https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation)